server/database/config/db_build.sql

```diff
@@ -33,22 +35,23 @@ CREATE TABLE course (
 
 CREATE TABLE dates (
 	id SERIAL PRIMARY KEY,
-    h_from INTEGER NOT NULL,
-    h_to INTEGER NOT NULL,
+    fromm INTEGER NOT NULL,
+    too INTEGER NOT NULL,
```

Comment:
@AbdallahAmmar96 Hi, I know these from / to fields weren't added in your PR, just renamed, but it might be a good time to change them from `INTEGER` to `DATE` or `DATETIME` now, to make those fields easier to work with.